state/index.js

```diff
@@ -1,9 +1,17 @@
+import fetch from 'node-fetch';
+import Promise from 'bluebird';
```

Comment:
Since we pulled in async/await transpiling, we can do something like:

```
const getSiteState = async () => ({
  featured_blog_posts: await blogFetcher.getFeaturedPosts(),
  ...state
});
```

Very sleek!